offchain-modules/cli/src/commands/types.rs

```diff
@@ -56,6 +57,24 @@ pub struct CreateBridgeCellArgs {
     pub simple_typescript: bool,
 }
 
+#[derive(Clap, Clone, Debug)]
+pub struct RecycleBridgeCellArgs {
+    #[clap(long, default_value = "~/.force-bridge/config.toml")]
+    pub config_path: String,
+    #[clap(long)]
+    pub network: Option<String>,
+    #[clap(short = 'k', long)]
+    pub private_key_path: Option<String>,
+    #[clap(long)]
+    pub private_key: Option<String>,
```

Comment:
why there are two private_key fields?

Reply:
private_key_path : the index of private key in the config file 
private_key: the private key string

because the bridge cell i offered by other private key which is not in recorded the config file

Reply:
> why there are two private_key fields?

private_key_path : the index of private key in the config file 
private_key: the private key string

because the bridge cell i offered by other private key which is not in recorded the config file

Reply:
private_key_path can support two kind params: index of privkey and path to a privkey file